docs/source.rst

```diff
@@ -4,73 +4,127 @@ Source Guide
+
+While using the Tor Browser on your personal computer helps hide your activity
+on the network, it leaves traces of its own installation on your local
+machine. For even greater deniability and security, we recommend booting into the
+`Tails operating system`_ (typically from a USB stick). Tails is specifically
+designed to run on your computer without leaving traces of your activity, and
+automatically routes all of your Internet browsing through Tor so you can easily
+access SecureDrop safely.
 
-Each Source Interface is only available as a *Tor Hidden Service*, which is a special type of website with an address ending in ".onion" that is only accessible through Tor. Tor is an anonymizing network that makes it difficult for anybody observing the network to associate a user's identity (e.g. their computer's IP address) with their activity (e.g. uploading information to SecureDrop).
+.. _`Tor Project website`: https://www.torproject.org/
+.. _`Tails operating system`: https://tails.boum.org/
 
-The easiest and most secure way to use Tor is to download the Tor Browser from the `Tor Project website`_. The Tor Browser is a modified version of the Firefox web browser designed to protect your security and anonymity while using Tor. If there is a chance that downloading the Tor Browser raises suspicion, you have a few alternatives.
+Choose Who to Submit To
+-----------------------
+We recommend conducting all research related to your submission in Tor Browser.
+If you are unsure whether you are using Tor, you can visit the address
+https://check.torproject.org.
 
-* If your mail provider is less likely to be monitored, you can send a mail to gettor@torproject.org and a bot will answer with instructions
-* If you routinely use GitHub, you can use it to `download the Tor Browser <https://github.com/TheTorProject/gettorbrowser>`__
+All organizations operating SecureDrop have a *Landing Page* that provides their
+own organization-specific recommendations for using SecureDrop. We encourage you
+to consider an organization's *Landing Page* before submitting to them.
 
-Once you have the Tor Browser, launch it and enter the ".onion" address for the Source Interface of the organization that you wish to submit to. You can find this address on the organization's *Landing Page*, or listed on the SecureDrop Directory.
+.. note::
 
-While using the Tor Browser on your personal computer helps hide your activity on the network, it leaves traces (of its own installation) on your local machine. For even more deniability, we recommend booting into a live system such as `Tails`_ for a higher level of security. Tails is specifically designed to run on your computer without leaving traces of your activity, and automatically routes all of your Internet browsing through Tor so you can easily access SecureDrop safely.
+   Each SecureDrop instance is totally independent, and submissions to that
```

Comment:
I propose "each SecureDrop instance is operated and administered independently by the news organization"

Reply:
Worded very similarly in  53e978e3a3ee5bfbdb81c2f144c00b4f60edb73b (I try not to overuse "news organization" since not all orgs using SecureDrop are strictly speaking news orgs).

Reply:
"host organization" feels like a fit, perhaps? The language "instance" and "host" both used throughout, is what felt best to me when I took a stab at this a while ago.